pylint_django/tests/input/func_noerror_foreign_key_in_non_django_class.py

```diff
@@ -0,0 +1,20 @@
+"""
+Checks that Pylint doesn't raise an error when a 'ForeignKey' appears in a
+non-django class
+
+The real case is described as follow:
+The project use tastypie and django.
+tastypie has a `ForeignKey` field which has the same name
+as django's `ForeignKey`.
+The issue is the lint trys resolving the `ForeignKey` for the
+tastypie `ForeignKey` which cause import error.
+"""
+# pylint: disable=missing-docstring
+from tastypie.resources import ModelResource
+from tastypie import fields
+from tastypie.fields import ForeignKey
+
+
+class MyTestResource(ModelResource): # pylint: disable=too-few-public-methods
+    field1 = ForeignKey('myapp.api.resource', 'xxx')
+    field2 = fields.ForeignKey('myapp.api.resource', 'xxx')
```

Comment:
The convention here is that such files are named `external_<dependency_name>_<scenario_name>.py`, for example `external_tastypie_noerror_foreign_key.py`

Reply:
Oh, thanks for the reminder, fixed it.